packages/api/src/libs/utils.ts

```diff
@@ -40,14 +45,19 @@ export function calcGcr(
 }
 
 export function calcTvl(
+  // price needs to be in wei
   price: string,
   emp: Pick<uma.tables.emps.Data, "totalPositionCollateral" | "collateralDecimals">
 ) {
   const { totalPositionCollateral, collateralDecimals } = emp;
   assert(uma.utils.exists(totalPositionCollateral), "requires total position collateral");
   assert(uma.utils.exists(collateralDecimals), "requires collateralDecimals");
   const normalizedCollateral = uma.utils.ConvertDecimals(collateralDecimals, 18)(totalPositionCollateral);
-  return parseUnits(price).mul(normalizedCollateral).div(parseUnits("1"));
+  return BigNumber.from(price).mul(normalizedCollateral).div(ETH);
```

Comment:
nit: why the name ETH? Might be a little confusing since it seems unrelated to the currency ETH? Thoughts on `SCALING_CONSTANT` or `SCALING_MULTIPLIER` instead?